lib/backend/test/suite.go

```diff
@@ -52,6 +53,11 @@ type ConstructionOptions struct {
 	// create an entirely independent data store, but instead should create a
 	// new interface to the same underlying data store as `ConcurrentBackend`.
 	ConcurrentBackend backend.Backend
+
+	// FastTTLDeletion indicates ttl deletions happen within a short time period
+	// for testing. Some backends may do async deletion that takes longer, too long
+	// to wait for testing deletion event propigation.
+	FastTTLDeletion bool
```

Comment:
Instead of skipping those tests we could also have a method implemented for the backend in a `_test.go` file that forces a synchronous TTL deletion. Especially when paired with a non-real clock, it could make the backend a bit more testable - with the caveat that actual TTL deletion might take much longer, in the wild.